src/uct/ib/mlx5/dc/dc_mlx5_ep.c

```diff
@@ -1743,7 +1743,7 @@ void uct_dc_mlx5_ep_handle_failure(uct_dc_mlx5_ep_t *ep,
 
     ucs_assert(!uct_dc_mlx5_iface_is_policy_shared(iface));
 
-    uct_dc_mlx5_update_tx_res(iface, txwq, txqp, pi);
+    uct_dc_mlx5_update_tx_res(iface, dci_index, pi);
```

Comment:
can we remove txqp, txwq variables?
seems could also remove UCT_DC_MLX5_IFACE_TXQP_DCI_GET macro altogether